baby-gru/src/components/validation-tools/MoorhenCarbohydrateValidation.tsx

```diff
@@ -0,0 +1,182 @@
+import { Col, Row, Form, Button } from 'react-bootstrap'
+import { MoorhenMapSelect } from '../select/MoorhenMapSelect'
+import { MoorhenMoleculeSelect } from '../select/MoorhenMoleculeSelect'
+import {PrivateerResultsEntry} from "../../types/privateer"
+import {setHoveredAtom} from "../../moorhen"
+interface Props extends moorhen.CollectedProps {
+    dropdownId: number;
+    accordionDropdownId: number;
+    setAccordionDropdownId: React.Dispatch<React.SetStateAction<number>>;
+    sideBarWidth: number;
+    showSideBar: boolean;
+}
+
+function PrivateerValidation(props: {data: PrivateerResultsEntry[], selectedModel: any, clickCallback: (e) => void}) {
+
+    const molecules = useSelector((state: any) => state.molecules);
+    async function handleClick(e) {
+        const newCenterString =
+            e.target.dataset.chainid +
+            '/' +
+            e.target.dataset.seqnum +
+            '(' +
+            e.target.dataset.resname +
+            ')';
+
+        props.clickCallback(newCenterString);
+        const selectedMolecule = molecules.find(
+            (molecule) => molecule.molNo === props.selectedModel
+        );
+        const _center = await selectedMolecule.centreOn(newCenterString);
+    }
+
+    const sugarRef = useCallback(
+        (node: HTMLElement | null) => {
+            if (node !== null) {
+                node.querySelector('svg').style.display = 'block';
+                node.querySelector('svg').style.margin = 'auto';
+
+                const useList = node.querySelectorAll('use');
+
+                for (let i = 0; i < useList.length; i++) {
+                    useList[i].addEventListener('click', handleClick);
+                    useList[i].addEventListener('mousedown', (e) => {
+                        e.stopPropagation();
+                    });
+                    useList[i].addEventListener('touchstart', (e) => {
+                        e.stopPropagation();
+                    });
+                }
+            }
+        }, []
+    );
+
+    return <>
+        {props.data.map((item, index) => {
+            return (
+                <div style={{padding: "0 0 1rem 1rem", borderBottom: "solid"}}>
+                    <div style={{display: "flex", justifyContent: "space-between", padding: "1rem 1rem 0rem 1rem"}}>
+                    <p>Glycan ID: {item.id}</p>
+                        <p>GlyToucan ID: {item.glytoucan_id}</p>
+                            <p>GlyConnect ID: {item.glyconnect_id}</p>
+                    </div>
+                <div
+                    style={{display: "flex", padding: "1rem"}}
+                    id="svgContainer"
+                    dangerouslySetInnerHTML={{
+                        __html: item.svg,
+                    }}
+                    ref={sugarRef}
+                />
+                </div>
+            )
+        })}
+
+        <p style={{margin: "auto"}}><i>Powered by Privateer</i></p>
```

Comment:
Rather than having this here, let's instead call the window modal "Carbohydrate validation with Privateer"